nexus/auth/src/authn/external/session_cookie.rs

```diff
@@ -199,26 +204,32 @@ mod test {
     use async_trait::async_trait;
     use chrono::{DateTime, Duration, Utc};
     use http;
+    use omicron_uuid_kinds::ConsoleSessionKind;
+    use omicron_uuid_kinds::TypedUuid;
```

Comment:
continuing from above:
```suggestion
    use omicron_uuid_kinds::ConsoleSessionUuid;
```
although, if you imported `super::*`, wouldn't need this...